provisioner/secret.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var base = require('taskcluster-base');
+var assert = require('assert');
+var debug = require('debug')('aws-provisioner:Secret')
+var taskcluster = require('taskcluster-client');
+
+var Secret = base.Entity.configure({
+  version: 1,
+
+  // TODO: verify these are valid azure keys - heeeelp please Jonas
+  partitionKey: base.Entity.keys.StringKey('token'),
+  rowKey: base.Entity.keys.StringKey('provisionerId'),
+
+  properties: {
+    token: base.Entity.types.String,
+    created: base.Entity.types.Date,
+    provisionerId: base.Entity.types.String,
+    workerType: base.Entity.types.String,
+    secrets: base.Entity.types.JSON,
+  },
```

Comment:
More than `created` I would like one called `expiry`... Because it is at creation we've decided how long time the spot request is valid for... And this must expire `1 hour` after the spot request has expired...

I don't recall if we do this already, but I'm pretty sure we planned to put an expiration in our spot requests...